beyond-essence/inventory/error-messages.md

```diff
@@ -0,0 +1,67 @@
+
+**Be specific**: Whenever possible, use a specific error message for each use case to be as helpful for the merchant as possible. Try to avoid generic messages like "Something went wrong" (What went wrong? Why? Is there anything the merchant can do to make it right?) or "The entered value is invalid" (Why is it invalid? What are the requirements for a valid entry?). 
+If a message really needs to be generic, because it needs to be used for different use cases or because the error can't be defined any further, write it at least as friendly and helpful as possible.
+
+**Be concise**: Error messages appear in boxes with restricted space. Also, when the merchant receives an error message, they are in the middle of performing an action that is being interrupted by the error. They want to resume their work as quickly as possible and don't want to read lengthy explanations, apologies or instructions.
+Brevity should not keep the message from sounding friendly and helpful, though. Be concise, but don't sound like a robot.
+
+**Be friendly & positive**: Instead of telling the merchant what they can *not* do, be positive and tell them what they *can* do.
+Even when the message needs to be short, don't make it sound threatening or commanding and don't make the merchant feel bad about having made a mistake.
+
+DO: You can add max. %{count} products.
+
+DON'T: You can't add more than %{count} products.
+
+DO: Incorrect password. Maybe you mistyped?
+
+DON'T: You've entered an incorrect password! Enter the correct password without typos.
+
+**Use humour only where appropriate**: Just as the merchant does not want to read through lengthy instructions when encountering an error, they do not want to be made fun of or see the same joke over and over in error messages that may appear more frequently.
+Humour should be very well dosed and never take the situation lightly. When in doubt, don't use humour. Also: Clarity and usability first. Don't use language or references that might be witty but prevent any of our merchants from understanding the message.
+
+MAYBE: *"I have a feeling I've seen this email before... would you like to log in instead of signing up?"*
+
+RATHER NOT: *We were unable to save your changes. But the force will be with you. Always.*
+
+NEVER: *"Oh no. All of your products have been deleted. Well, that's not the end of the world, is it?"*
+
+## Use cases
+
+In the Beyond cockpit we currently use two types of error messages: Toast messages and Validation messages.
+
+**[Toast messages](/beyond-essence/inventory/toast-messages/)**: For errors that affect the entire workflow. These messages are displayed until the error is resolved. Most of the error messages shown in toast messages are currently unspecific internal error messages.
```

Comment:
this isn't always true in the current system 😬 , some flash for 8 seconds (I think). Also I might reconsider this sentence when you think about system errors that cannot be resolved ("contact Support with the following identifier..."- they stay until clicked away, not until they are resolved (maybe they would if they didn't close them...)

Reply:
I think I took this sentence from another page in essence: https://developer.epages.com/beyond-essence/inventory/toast-messages/
But if this is not true, I think it would also be ok to just remove this sentence here, especially considering this links to the page about toast messages. What do you think?

Reply:
yes, I'm okay with that. If we're focusing here just on copywriting standards, we don't have to get into the specifics of how each message-type behaves (particularly when it's somewhat inconsistent ;)